### ISSUE TITLE
[BUG] Timeout when scraping game data

### ISSUE THREAD

**Issue by grez1** [Depressurizer/Depressurizer#146]
Tools/Database/Scrape Unscraped

During the run, the engine will make it through anywhere between 15 and 150 games (I have a very large library, ~4500), getting slower and slower as it goes. Finally, it will pause and then timeout. The behavior is consistent. 

If I attempt to restart the process, it will work ~50% of the time if it's the first time I've loaded the app after a reboot or the like. After that, restarting the process will just hang and timeout immediately. The only way to get it going again is to save and exit the app entirely, reload, and continue the process, at which point it gets through another 20 or 30 games, and then times out again.

At this rate, it's going to take me a week to scrape all of my data. :D </salty>

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro 64-bit
 - Depressurizer 4.10
 -  .NET Version 4.7.03056

Hardware is AMD TR2 2950X, Gigabyte X399 Aorus xtreme 

Thanks for the help!

**Comment by Gaffi** [comment]
@grez1 
I believe this is due to removed games or adult/age check games. It was a bit of a process, but I manually removed all categories, dumped every title into a new category (called "Unscraped", but whatever), and scraped through. As items were categorized, I removed them from the "Unscraped" category. The process kept hanging along the way, so it took many attempts to get through them all. Because of that, I scraped in batches of 50-100 games at a time.

When done, I was left with quite a few items that I recategorized as "Not Scrapable." Although I didn't exhaustively check all of the games in that list, the *vast* majority were removed items or things that couldn't be viewed without logging in to Steam first. From this point on, any scraping has worked just fine. In the event a game gets removed from the store, I can move it to the "Not Scrapable" category, and I expect it should continue fine.

I'm pinging here to hopefully help get the bug resolved, but maybe this method works for you?
